taichi/rhi/opengl/opengl_device.cpp

```diff
@@ -546,10 +538,6 @@ DeviceAllocation GLDevice::allocate_memory(const AllocParams &params) {
     target_hint = GL_SHADER_STORAGE_BUFFER;
   } else if (params.usage && AllocUsage::Uniform) {
     target_hint = GL_UNIFORM_BUFFER;
-  } else if (params.host_write && params.host_read) {
```

Comment:
nit: why do we remove these btw?

Reply:
They don't actually modify the hint right now.